plugins/modules/ec2_instance.py

```diff
@@ -55,6 +57,17 @@
         Only required when instance is not already present.
     default: t2.micro
     type: str
+  count:
+    description:
+      - Number of instances to launch.
```

Comment:
There should probably be a note in here that setting this value will result in **always** creating new instances.